velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,40 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add :doc:`json_extract_scalar <../functions/json>` function.
```

Comment:
This sounds like we added json_extract_scalar function, not that we adding documentation for the existing function. Perhaps, clarify as "Add missing documentation for :func:`json_extract_scalar` function". Also, using :func: for functions allows for linking to the anchor on the page where the function is defined.